manifester_settings.yaml.example

```diff
@@ -0,0 +1,36 @@
+#rhsm-manifester settings
+log_level: "info"
+offline_token: ""
+proxies: {"https": ""}
+manifest_category:
+  golden_ticket:
+    # An offline token can be generated at https://access.redhat.com/management/api
+    offline_token: ""
+    # Value of sat_version setting should be in the form 'sat-6.10'
+    sat_version: "sat-6.10"
```

Comment:
6.12 for master branch

Reply:
Required: format this using `self.server.version`

Reply:
While I take your point that the version number should not be hardcoded, I'm not sure exactly what you're referring to with `self.server.version`. This pattern doesn't appear to be used elsewhere in Robottelo. I see a couple of uses of `settings.server.version.release` - is this what you were referring to, or did you mean something else?

Reply:
Its a `this.server.version` reference actually. Best upstream example is in the capsule template:

https://github.com/SatelliteQE/robottelo/blob/master/conf/capsule.yaml.template#L3

This one is using `robottelo.satellite_version`, but I would recommend using `server.version` fields instead.

Reply:
Ok, I'm getting errors with both of these. I've tried a couple of different variants, including:

`sat_version: "@jinja sat-{{this.server.version}}"`
`sat_version: "@format sat-{this.server.version}"`
`sat_version: "@jinja sat-{{this.robottelo.satellite_version}}"`
`sat_version: "@format sat-{this.robottelo.satellite_version}"`

They're all resulting in errors when the expression in braces is evaluated. Is this expected to work when running locally? Might the fact that I have manifester_settings.yaml in the base robottelo directory instead of in `robottelo/conf/` be a factor?

Reply:
_conf/manifester_settings.yaml_:
```yaml
MANIFESTER:
  manifest_category:
    robottelo_automation:
      sat_version: "@jinja sat-{{this.server.version.release}}"
```

```python
from robottelo.config import settings
settings.manifester.manifest_category.robottelo_automation.sat_version
```

Gives me `sat-6.11`